src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -67,56 +70,94 @@
      * ride or kiss and ride). By default limited to 30 minutes driving, because if it's unlimited on
      * large graphs the search becomes very slow.
+     *
+     * @Deprecated - Not currently supported in Raptor, but planned in the future
      */
+    @Deprecated
     public int maxPreTransitTime = 30 * 60;
 
-    /** The worst possible time (latest for depart-by and earliest for arrive-by) to accept */
+    /** The worst possible time (latest for depart-by and earliest for arrive-by) to accept
+     *
+     * @Deprecated - Not currently supported in Raptor, but planned in the future
+     * */
+    @Deprecated
     public long worstTime = Long.MAX_VALUE;
 
-    /** The worst possible weight that we will accept when planning a trip. */
+    /** The worst possible weight that we will accept when planning a trip.
+     *
+     * @Deprecated - Not supported in Raptor
+     * */
+    @Deprecated
     public double maxWeight = Double.MAX_VALUE;
 
-    /** The maximum duration of a returned itinerary, in hours. */
+    /** The maximum duration of a returned itinerary, in hours.
+     *
+     * @Deprecated - Not supported in Raptor
+     * */
+    @Deprecated
     public double maxHours = Double.MAX_VALUE;
 
-    /** Whether maxHours limit should consider wait/idle time between the itinerary and the requested arrive/depart time. */
+    /** Whether maxHours limit should consider wait/idle time between the itinerary and the requested arrive/depart time.
+     *
+     * @Deprecated - Not supported in Raptor
+     * */
+    @Deprecated
     public boolean useRequestedDateTimeInMaxHours = false;
 
     /** The set of TraverseModes that a user is willing to use. Defaults to WALK | TRANSIT. */
     public TraverseModeSet modes = new TraverseModeSet("TRANSIT,WALK"); // defaults in constructor overwrite this
 
-    /** The set of characteristics that the user wants to optimize for -- defaults to QUICK, or optimize for transit time. */
+    /** The set of characteristics that the user wants to optimize for -- defaults to QUICK, or optimize for transit time.
+     *
+     * @Deprecated - Not supported in Raptor
+     * */
+    @Deprecated
     public OptimizeType optimize = OptimizeType.QUICK;
-    // TODO this should be completely removed and done only with individual cost parameters
     // Also: apparently OptimizeType only affects BICYCLE mode traversal of street segments.
     // If this is the case it should be very well documented and carried over into the Enum name.
 
     /** The epoch date/time that the trip should depart (or arrive, for requests where arriveBy is true) */
     public long dateTime = new Date().getTime() / 1000;
 
-    /** Whether the trip should depart at dateTime (false, the default), or arrive at dateTime. */
+    /** Whether the trip should depart at dateTime (false, the default), or arrive at dateTime.
+     *
+     * TODO: Needs to  be implemented
+     * */
+    @Deprecated
```

Comment:
Ideally we should support an enum: DEPART_FROM, ARRIVE_BY, and TIME_TABLE - but I guess we can emulate this by a combination of this flag and setting the from/to times.